assets/src/components/monitor/server/parts/body/processes-list/program.tsx

```diff
@@ -22,7 +21,10 @@ export const Program = ({ group, server }: { group: ApiProcessGroup; server: Api
     <div className='px-2 py-1'>
       <div className='flex items-center space-x-2'>
         <div>
-          <div className='flex cursor-pointer items-center space-x-2 program-title' onClick={() => setIsActive(!isActive)}>
```

Comment:
Did the string exceed 120 characters (or whatever the limit is, I could be wrong)?

Reply:
It's normal for base prettier limit <200 chars